docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -726,3 +726,73 @@ Tasks are not running, and the server logs contain the following error message:
+You should take a look at the Kibana log and see what you can find that relates to Task Manager.
+In a healthy environment you should see a log line that indicates that Task Manager was successfully started when Kibana was:
+[source, txt]
+--------------------------------------------------
+server log [12:41:33.672] [info][plugins][taskManager][taskManager] TaskManager is identified by the Kibana UUID: 5b2de169-2785-441b-ae8c-186a1936b17d
+--------------------------------------------------
+
+If you see that message and no other errors that relate to Task Manager, it’s most likely that Task Manager is running fine and has simply not had the chance to pick the task up yet.
+If, on the other hand, the runAt is severely overdue, then it’s worth looking for other Task Manager or Alerting related errors, as something else may have gone wrong.
+It’s worth looking at the status field, as it might have failed, which would explain why it hasn’t been picked up or it might be running which means the task might simply be a very long running one.
+
+[float]
+[[task-marked-failed]]
+==== What do I do if the Task is marked as failed?
+
+*Problem*:
+
+Tasks marked as failed.
+
+*Solution*:
+
+Broadly speaking the Alerting framework is meant to gracefully handle the cases where a task is failing by rescheduling a fresh run in the future. If this fails to happen, then that means something has gone wrong in the underlying implementation and this isn’t expected.
+Ideally you should try and find any log lines that relate to this rule and its task, and use these to help us investigate further. Don’t forget to include the state of the rule (by using the http endpoints) in any ticket you submit.
+
+[float]
+[[task-manager-kibana-log]]
+==== Task Manager Kibana Log
+Task manager will write log lines to the Kibana Log on certain occasions. Below are some common log lines and what they mean.
+
+Task Manager has run out of Available Workers:
+[source, txt]
+--------------------------------------------------
+server log [12:41:33.672] [info][plugins][taskManager][taskManager] [Task Ownership]: Task Manager has skipped Claiming Ownership of available tasks at it has ran out Available Workers.
+--------------------------------------------------
+
+This log message tells us that Task Manager is not managing to keep up with the sheer amount of work it has been tasked with completing. This might mean that Alerts are not running at the frequency that was expected (instead of running every 5 minutes, it runs every 7-8 minutes, just as an example).
+
+By default Task Manager is limited to 10 tasks and this can be bumped up by setting a higher number in the kibana.yml file using the xpack.task_manager.max_workers configuration. That said, though, it’s important to keep in mind that a higher number of tasks running at any given time means more load on both Kibana and Elasticsearch, so only do so if adding this additional load on the environment makes sense.
```

Comment:
```suggestion
By default Task Manager is limited to 10 tasks and this can be bumped up by setting a higher number in the kibana.yml file using the `xpack.task_manager.max_workers` configuration. It is important to keep in mind that a higher number of tasks running at any given time means more load on both Kibana and Elasticsearch, so only change this setting if increasing load in your environment makes sense.
```